english/sleep/more-sleep/expired-milk/tell-mom/tell-mom.md

```diff
@@ -0,0 +1 @@
+You tell mom about the milk. She instead scolds you for being careless. She is thankful though that you did not get food poisoning
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.